src/pybind/mgr/dashboard/services/rbd.py

```diff
@@ -60,13 +60,29 @@ def format_features(features):
     return res
 
 
+def get_image_spec(pool_name, namespace, rbd_name):
+    namespace = '{}/'.format(namespace) if namespace else ''
+    return '{}/{}{}'.format(pool_name, namespace, rbd_name)
+
+
+def parse_image_spec(image_spec):
+    namespace_spec, image_name = image_spec.rsplit('/', 1)
+    if '/' in namespace_spec:
+        pool_name, namespace = namespace_spec.rsplit('/', 1)
+    else:
+        pool_name, namespace = namespace_spec, None
+    return pool_name, namespace, image_name
+
+
 class RbdConfiguration(object):
     _rbd = rbd.RBD()
 
-    def __init__(self, pool_name='', image_name='', pool_ioctx=None, image_ioctx=None):
+    def __init__(self, pool_name='', namespace=None, image_name='', pool_ioctx=None,
+                 image_ioctx=None):
         # type: (str, str, object, object) -> None
         assert bool(pool_name) != bool(pool_ioctx)  # xor
         self._pool_name = pool_name
+        self._namespace = namespace if namespace is not None else ''
```

Comment:
`pool_name` and `image_name` have default empty string values.

Reply:
I've changed the default value to an empty string.